tox.ini

```diff
@@ -1,26 +1,25 @@
 [tox]
-envlist = py26, py27, py33, py34, py35, flake8
-
-[travis]
-python =
-    3.5: py35
-    3.4: py34
-    3.3: py33
-    2.7: py27
-    2.6: py26
-
-[testenv:flake8]
-basepython=python
-deps=flake8
-commands=flake8 pystac
+envlist = py34, py35, py36
 
 [testenv]
-setenv =
-    PYTHONPATH = {toxinidir}
 
-commands = python setup.py test
+commands =
+    check-manifest --ignore tox.ini
+    {envpython} setup.py check -m -r -s
+    #    flake8 .
+    {envpython} setup.py test
+
+deps =
+    # Numpy is a secondary dependency of rasterio, so it must be 
+    # manually installed in the virtualenv before running
+    # setup.py install.
+    # See https://github.com/numpy/numpy/issues/2434 for more 
+    # information
+    numpy
+    flake8
+    check-manifest
+    docutils
 
-; If you want to make tox run the tests with the same versions, create a
-; requirements.txt with the pinned versions and uncomment the following lines:
-; deps =
-;     -r{toxinidir}/requirements.txt
+[flake8]
+exclude = .tox,*.egg,build,data
+select = E,W,F
```

Comment:
Can you add `max-line-length: 100` -- the default is pretty small (80) and as a team that's what we default too